tsconfig.build.json

```diff
@@ -2,7 +2,8 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "sourceMap": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "outDir": "./dist"
```

Comment:
Why did you add this?

Reply:
this is fixing error in node_module files